src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -139,7 +121,10 @@ function ReportActionItemFragment(props) {
 
             return (
                 <Text style={[containsOnlyEmojis ? styles.onlyEmojisText : undefined, styles.ltr, ...props.style]}>
-                    {checkForEmojiForSelection(text, props.displayAsGroup)}
+                    <ZeroWidthComponent
```

Comment:
this is breaking web
<img width="665" alt="Screenshot 2023-10-20 at 16 37 48" src="https://github.com/Expensify/App/assets/28352309/0ba746a8-a0d6-4f0a-bb84-f801ff27f3fc">